repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ValidateError.scala

```diff
@@ -0,0 +1,3 @@
+package cromwell.pipeline.datastorage.dto
+
+case class ValidateError(message: String) extends Exception(message)
```

Comment:
We have package 'validator' for similar classes.
The same class exists in play.api.data.validation library, I don't sure in useful applying these libraries here, but we can discuss it @myazinn @ilyaonishenko

Reply:
validation in play.api is using for validate json forms. My error is to inform user that WDL file has incorrect syntax

Reply:
About `validator ` package - same question should be for `VersioningException` is `ProjectVersioning` trait then.

Reply:
As far as I can see this `ValidateError` has nothing to do with play-json.
`validator` has a different purpose, I think this exception should not be put into that package. @ilyaonishenko said that soon we will have separate modules for the custom exception. Since it doesn't exists now, I think it is okay to put an exception here.